### PR TITLE
SNO-202-python3.8.5-upgrade

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,14 +41,17 @@ commands:
       - run:
           name: Install dependencies
           command: |
+            sudo wget -qO - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo tee /etc/apt/pg_gpg_key
+            sudo apt-key add /etc/apt/pg_gpg_key
+            echo "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main" | sudo tee -a /etc/apt/sources.list
+            sudo apt-get update && sudo apt-get install -y postgresql-11
```
Comment (keenangraham) [inline .circleci/config.yml:47]: This line isn't in the snovault circle config?

File: cloud-config/run-scripts/app-pg.sh [hunk 2]
```diff
@@ -148,12 +148,17 @@ for filename in 'AWS_ACCESS_KEY_ID' 'AWS_SECRET_ACCESS_KEY' 'AWS_REGION'; do
 done
 copy_with_permission "$PG_CONF_DEST" "$WALE_ENV" 'WALE_S3_PREFIX'
 
+## Install python3.6 to run wal-e with
+# python3.6 is on the ami now
+# sudo add-apt-repository -y ppa:deadsnakes/ppa
+# sudo apt-get update
+# sudo apt-get install -y python3.6 python3.6-dev python3.6-venv
```
Comment (keenangraham) [inline cloud-config/run-scripts/app-pg.sh]: These are commented out?

File: cloud-config/run-scripts/ami-python.sh [hunk 3]
```diff
@@ -1,79 +0,0 @@
-#!/bin/bash
```
Comment (keenangraham) [inline cloud-config/run-scripts/ami-python.sh:1]: How is mod_wsgi installed now?

**Review by keenangraham — DISMISSED** [review]

File: cloud-config/run-scripts/app-pg.sh (hunk 2 above)
Comment (ottojolanki) [reply cloud-config/run-scripts/app-pg.sh]: yeah, I should actually just remove those

File: cloud-config/run-scripts/ami-python.sh (hunk 3 above)
Comment (ottojolanki) [reply cloud-config/run-scripts/ami-python.sh:1]: It is installed in the AMI from ubuntu package. Not needed for python 3.6.

File: .circleci/config.yml (hunk 1 above)
Comment (ottojolanki) [reply .circleci/config.yml:47]: https://github.com/ENCODE-DCC/snovault/pull/344/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R34 etc

File: cloud-config/run-scripts/app-pg.sh (hunk 2 above)
Comment (ottojolanki) [reply cloud-config/run-scripts/app-pg.sh]: Installed in the AMI, and forgot to remove.

**Review by keenangraham — DISMISSED** [review]

**Review by keenangraham — APPROVED** [review]

**Pull request merged**